common/logger/src/struct_log.rs

```diff
@@ -53,6 +54,8 @@ static FIELDS_TO_KEEP: &[&str] = &["error"];
 
 #[derive(Default, Serialize)]
 pub struct StructuredLogEntry {
+    /// Unique Id representing this message in Elasticsearch
+    id: String,
```

Comment:
how expensive is uuid generation?

Reply:
It's a `thread_rng` for 16 bytes.  I've basically copied this and removed the extra versioning and information that isn't really needed from the unique ID for this use case.

Reply:
One small improvement would be to avoid the extra heap allocation and make this a fixed length array of bytes (if you know that this will always be 16 bytes for example), but that's probably splitting hairs.